src/firefly/js/ui/DownloadOptionsDialog.jsx

```diff
@@ -52,29 +51,28 @@ export function DownloadOptionsDialog({fromGroupKey, children, fileName, labelWi
                 </div>
         );
 
+        const style = {
+                width: '100%',
+                height: '15rem',
+                paddingLeft: '1rem',
+                paddingRight: '1rem',
+                paddingTop: '1rem',
+                border: '1px solid #a3aeb9',
+                borderRadius: 5,
+                overflow: 'auto'
+            };
+
         const showSave = (
-            <div  style={{width: dialogWidth, height: dialogHeight}}>
-                <div style={{marginTop: 10,
-                             boxSizing: 'border-box',
-                             width: 'calc(100%)', height: 'calc(100% - 10px)',
-                             overflow: 'auto',
-                             padding: 5,
-                             border:'1px solid #a3aeb9'}}>
-                    <WorkspaceSave fieldKey={'wsSelect'} files={wsList} value={wsSelect}
-                        tooltip='workspace file system'/>
-                </div>
-            </div>
+            <Box style={style} sx={{flexGrow: 1, overflow: 'auto', mt: 3 }}>
```

Comment:
Why use `style` and `sx`?  `sx` can handle style as well.
- overflow is defined twice
- paddingLeft and paddingRight is just `px` in sx syntax.  e.g.  sx={{px:'1rem'}}
- should not be using border and custom border color.  use a JoyUI surface instead, like <Sheet variant='outlined'>
- height: '15rem' and flexGrow: 1 may not work.  It can't grow if height is fixed.